pkg/ref/handler.go

```diff
@@ -13,28 +14,45 @@ import (
 // Example:
 //   err = cnt.Watch(
 //      &source.Kind{
-//         Type: &api.Plan{},
+//         Type: &api.Referenced{},
 //      },
-//      libref.Handler())
-func Handler() handler.EventHandler {
+//      libref.Handler(&api.Owner{}))
+func Handler(owner interface{}) handler.EventHandler {
```

Comment:
The handler really needed to filter by owner.
Without this, controllers were getting reconcile events for _other_ CR types.